ludwig/data/preprocessing.py

```diff
@@ -1195,6 +1199,10 @@ def build_dataset(
         preprocessing_parameters = feature_name_to_preprocessing_parameters[feature_config[NAME]]
         handle_missing_values(dataset_cols, feature_config, preprocessing_parameters, backend)
 
+    # If we're using LLMs for zero-shot/few-shot learning, update text input features with the prompt
+    # ahead of time so that we can compute metadata and build the preprocessed data correctly.
+    handle_llm_prompt_injection(dataset_cols, feature_configs, feature_name_to_preprocessing_parameters, backend)
```

Comment:
prompt injection is the name of a technique for "hacking" llms. i would rename this

Reply:
Agreed! Just updated the names of the functions and variables to better reflect what the function is doing